content/guides/walkthroughs/run-hub-images.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Run Docker Hub images
+keywords: get started, quick start, intro, concepts
+description: Learn how to run Docker Hub images
+aliases:
+- /get-started/run-docker-hub-images/
+---
+
+You can share and store images in Docker Hub
+([http://hub.docker.com](http://hub.docker.com)). Docker Hub has over 100,000
+images created by developers that you can run locally. You can search for Docker
+Hub images and run them directly from Docker Desktop.
+
+Before you start, [get Docker Desktop](../../get-docker.md).
+
+## Step 1: Search for the image
+
+You can search for Docker Hub images on Docker Desktop. To search for the image used in this walkthrough, do the following:
+
+1. Open Docker Desktop and select the search.
+2. Specify `docker/welcome-to-docker` in the search.
+
+![Search Docker Desktop for the welcome-to-docker image](images/getting-started-search.png?w=400)
+
+## Step 2: Run the image
+
+To run the `docker/welcome-to-docker` image, do the following:
```

Comment:
Nit: do we need 'do the following'? we could just dive straight in with 'To run....image:'?

(I see this is repeated throughout the walkthroughs)